CODEOWNERS

```diff
@@ -3,3 +3,18 @@
 
 # For anything not explicitly taken by someone else:
 * @open-telemetry/dotnet-approvers
+
+src/OpenTelemetry.Contrib.Exporter.Stackdriver/                             @open-telemetry/dotnet-approvers
```

Comment:
I think this should be ```@open-telemetry/dotnet-contrib-approvers```?

Reply:
Since I'm not a member of the open-telemetry, I couldn't verify if the the `@open-telemetry/dotnet-contrib-approvers` has been created yet.
If it already exists or is going to be created soon, I can update these.